### DISCUSSION TITLE
Hello, how can I make the rag-api endpoint follow the Compatible AI Endpoints I set up?

### DISCUSSION THREAD

**Discussion by sunsky89757** [danny-avila/LibreChat#2295] [Troubleshooting]
### What is your question?

![image](https://github.com/danny-avila/LibreChat/assets/113254524/92feb860-f4cd-4def-8574-8804ae03dcf3)
Although I set the base_url of the Compatible AI to https://api.juheai.top/v1, rag-api still uses openai as the base_url. How can I change it?

### More Details

As described above

### What is the main subject of your question?

Endpoints

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/113254524/92feb860-f4cd-4def-8574-8804ae03dcf3)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
You have several options right now as shown on the readme on the RAG_API project: https://github.com/danny-avila/rag_api

however you cannot currently set an alternate base url. However this is on my list for today to add as it is relatively simple to add.

**Reply by sunsky89757** [reply]
Thank you very much, looking forward.
